website/src/index.md

```diff
@@ -5,7 +5,7 @@ layout: home
 hero:
   name: Tachiyomi
   text: Full-featured reader
-  tagline: Read your favorite manga, webtoons, comics, and more – easier than ever on your Android.
+  tagline: Discover and read manga, webtoons, comics, and more – easier than ever on your Android device.
```

Comment:
Need to change in the site config description as well, as it will reflect on the RSS description for example.

Reply:
This line is present in:
- [`README.md`](https://github.com/xhenos/kodo/blob/v3-vitepress/README.md)
- [`index.md`](https://github.com/xhenos/kodo/blob/v3-vitepress/website/src/index.md)
- [`config.ts`](https://github.com/xhenos/kodo/blob/v3-vitepress/website/src/.vitepress/config.ts)
- [`headConfig.ts`](https://github.com/xhenos/kodo/blob/v3-vitepress/website/src/.vitepress/config/headConfig.ts)